docs/references/expo/offline-support.mdx

```diff
@@ -71,3 +71,45 @@ To enable offline support in your Expo app, follow these steps:
   }
   ```
 </Steps>
+
+## Handling network errors
+
+To handle network errors in your Expo app, you can use the `isClerkRuntimeError` function alongside the `code` property to identify the specific error. Here's an example of how you can handle network errors:
+
+```tsx {{ mark: [1, [24, 26]] }}
+import { isClerkRuntimeError } from '@clerk/clerk-js'
+import { useSignIn } from '@clerk/clerk-expo'
+import { Link, useRouter } from 'expo-router'
+import { Text, TextInput, Button, View } from 'react-native'
+import React from 'react'
+
+export default function SignInPage() {
+  const { signIn, setActive, isLoaded } = useSignIn()
+  const router = useRouter()
+
+  const [emailAddress, setEmailAddress] = React.useState('')
+  const [password, setPassword] = React.useState('')
+
+  const onSignInPress = React.useCallback(async () => {
+    if (!isLoaded) return
+    try {
+      const signInAttempt = await signIn.create({
+        identifier: emailAddress,
+        password,
+      })
+
+      // more logic here
```

Comment:
```suggestion
      // logic to handle errors 
```

Reply:
We could remove this comment if it's confusing. 🤷